e2e/utils/command-utils.ts

```diff
@@ -70,26 +71,33 @@ export function runCommand(
 ): string {
   const { failOnError, ...childProcessOptions } = options ?? {};
   try {
-    const r = execSync(command, {
+    const r = execSync(`${command}${isVerbose() ? ' --verbose' : ''}`, {
```

Comment:
We need to be careful with this since it changes the output from the command.

Reply:
Yes, but I think it's fair to change the output. We do say "if you want to get a full log run with verbose" and I think that it makes sense to add the `--verbose` to these commands directly.